src/answers-umd.js

```diff
@@ -418,6 +418,61 @@ class Answers {
     this.coreAdapter.storage.setWithPersist(StorageKeys.QUERY, query);
   }
 
+  /**
+   * Conduct a vertical search with the provided options.
+   *
+   * @param {Object} options
+   * @param {string} options.verticalKey the vertical ID for the search
+   * @param {string} options.query the query string of the search
+   * @param {boolean} options.useFacets enables facets if true
+   * @param {boolean} options.resetPagination resets the search offset if true
+   * @param {string} options.queryId the queryId associated with the search
+   * @param {boolean} options.appendQuery if true, adds the results of this query to the current results
+   */
+  verticalSearch (options) {
+    const verticalKey = options.verticalKey;
+    const verticalOptions = {
+      ...options.useFacets && { useFacets: options.useFacets },
+      ...options.resetPagination && { resetPagination: options.resetPagination }
+    };
+    const queryOptions = {
+      ...options.query && { input: options.query },
+      ...options.queryId && { id: options.queryId },
+      ...options.appendQuery && { append: options.appendQuery }
+    };
+    this.coreAdapter.verticalSearch(verticalKey, verticalOptions, queryOptions);
+  }
+
+  /**
+   * Rests all filters in the filter registry.
+   */
+  resetAllFilters () {
+    const filterNodes = this.coreAdapter.filterRegistry
+      .getAllFilterNodes()
+      .filter(fn => fn.getFilter().getFilterKey());
+    filterNodes.forEach(node => {
+      node.remove();
+    });
+  }
+
+  /**
+   * Retrieve data from storage.
+   *
+   * @param {string} storageKey
+   */
+  getFromStorage (storageKey) {
```

Comment:
This feels like we should support all CRUD operations. I have `delete` and `set` in the InteractiveMap operations. Is this something we're thinking about adding?

Reply:
ya, if we have a use case for it I think we should add it here. What do you think of the names 'deleteFromStorage' and 'setStorage'?